src/main/java/Daniela_Chihai/com/Endava/SoA5/Java_Basics/Task2_1.java

```diff
@@ -0,0 +1,20 @@
+package Daniela_Chihai.com.Endava.SoA5.Java_Basics;
+
+import java.util.Scanner;
+
+public class Task2_1 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter 2 numbers which would represent the start and the end range: ");
```

Comment:
In this case should be entered only one number 😃

Reply:
Thank you.